lib/evaluateFunctions.js

```diff
@@ -382,26 +377,30 @@ function evaluateNthRootForChildren(node) {
     }
   }
 
-  if (substeps.length > 0) {
+  if (substeps.length === 0) {
+    return NodeStatus.noChange(node);
+  }
+  else if (substeps.length === 1) {
+    return substeps[0];
+  }
+  else {
     return NodeStatus.nodeChanged(
       MathChangeTypes.EVALUATE_DISTRIBUTED_NTH_ROOT, node, newNode, true, substeps);
   }
-
-  return NodeStatus.noChange(node);
 }
 
 // Given a multiplication nodes, including nthRoots, will combine the nodes
```

Comment:
given a root node with multiplication nodes? or just multiplication nodes. still not super clear cause the first thing you do is getRootNode on the node..